internal/account/idgenerator.go

```diff
@@ -0,0 +1,24 @@
+package account
+
+import (
+	"crypto/rand"
+	"math/big"
+	"strings"
+)
+
+const charset = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+
+func GenerateRandomID() (string, error) {
```

Comment:
I think we could do that for now, since it's not used anywhere yet
it helps to improve clarity - "oh it's going to only be used on this package"
```suggestion
func generateRandomID() (string, error) {
```